packages/emoji/emoji.1.0.0/descr

```diff
@@ -0,0 +1,13 @@
+Use emojis by name
+
+Names for byte code sequences of common emojis
+
+#require "emoji";;
```

Comment:
It's rendered onto opam.ocaml.org as well; e.g. https://opam.ocaml.org/packages/topkg/